### PR TITLE
[WIP] Fix TRAVIS_BRANCH

### PR DESCRIPTION
**Pull request by bdunne**
- This appears to be blank on master

### CODE REVIEW

**Comment by Fryguy** [comment]
I think it should be ref_name... That's what I was checking in my pr

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -32,5 +32,5 @@ jobs:
     - name: Run container build script
       run: bin/build_container_image
       env:
+        TRAVIS_BRANCH: ${{ github.ref }}
```
Comment (Fryguy) [inline .github/workflows/ci.yaml:35]: ref won't work because ref == "refs/heads/master" on a regular branch.

**Comment by Fryguy** [comment]
I opened #240 with my branch (we were doing it at the same time)

**Comment by bdunne** [comment]
Closing in favor of #240

**Pull request closed without merging**